frontend/src/packages/drivers/drivers-api.ts

```diff
@@ -21,12 +20,11 @@ class DriverApi extends HttpApi {
     super({ path: ApiPath.BUSINESS, baseUrl, http, storage });
   }
 
-  public async getPageOfDrivers({
-    page,
-    size,
-  }: GetPaginatedPageQuery): Promise<DriverGetAllResponseDto> {
+  public async getPageOfDrivers(
+    queryString?: string,
+  ): Promise<DriverGetAllResponseDto> {
```

Comment:
```suggestion
type QueryParameters = { query?: string };

  public async getPageOfDrivers({ query = '' }: QueryParameters ): Promise<DriverGetAllResponseDto> {
```
 this.getFullEndpoint(`${ApiPath.DRIVERS}?${queryString}`, {}),